docs/data/base/getting-started/overview/overview.md

```diff
@@ -0,0 +1,23 @@
+# MUI Base - Overview
+
+<p class="description">MUI Base is a library of blank ("unstyled") React UI components and low-level hooks, ideally suited for implementing a custom design system.</p>
+
+## Introduction
+
+MUI Base is library of unstyled React UI components.
+It features the same robust engineering as Material UI, but without implementing Material Design.
+
+With MUI Base, developers can rapidly build UIs with custom design systems, without the need to override any default style settings.
+
+## Why use MUI Base?
+
+MUI Base is ideally suited for developers working with a custom design system.
```

Comment:
Not sure if we add value by mentioning what the "ideal" use case is... If I read this, as a user, I'd probably feel like I'm doing something wrong for using Base for something else than building a design system, which would be a perfectly acceptable use case.

Reply:
Rather than a single ideal, do you think we should make a list of potential use cases? I think that could give us an opportunity to further differentiate it from Material UI.

Reply:
For sure, that's a great idea!